JS/Planes/Plane.js

```diff
@@ -1,28 +1,23 @@
 class Plane {
+  constructor(model, maxSpeed, maxFlightDistance, maxLoadCapacity) {
+    this.basePlane = { model, maxSpeed, maxFlightDistance, maxLoadCapacity };
```

Comment:
What are you doing with this code?
You should pass one single argument to your constructor, but it should be an oblect. Please correct.